### PR TITLE
BookA-Pset_InSituTestHydro (previously) Pumping

### PR DESCRIPTION
**Pull request by larswik**
# Pset: Pset_InSituTestHydro

## Description: 

Properties for the results of in-situ pumping tests in soil or rock.

| Property | Datatype | Description |
|----------|----------|-------------|
|HydraulicPermeability|IfcLinearVelocityMeasure|A measurement of the ability of a fluid to flow through a rock. Represented by the variable "k". The quality of the soil that enables water to move through the profile. Permeability is measured as the number of metres per sec that water moves through the saturated soil. |
|HydraulicConductivity| According to existing definition | According to existing definition. |
|Transmissivity|IfcReal|Rate of flow of groundwater under a unit hydraulic gradient through an aquifer of unit width and unit thickness. That is, transmissivity is the product of hydraulic conductivity and thickness of the aquifer. The unit is in m2/second.|
|Storativity|IfcPositiveRatioMeasure|Volume of water released from storage per unit surface area of a confined aquifer per unit decline in hydraulic head. It is dimensionless.|
|LugeonValue|IfcReal|Lugeon Value = (q / L) x (P0 / P) q - flow rate [lit/min] L - Length of the borehole test interval [m] P0 - reference pressure of 1 MPa [MPa] P - Test pressure [MPa]|

### CODE REVIEW

**Comment by patmclarin** [comment]
please update the Pset description to: Properties for the results of in-situ pumping tests in soil or rock

File: IFC4x3/Properties/h/HydraulicPermeability_3iagtDsw9DTgUKGeH1bJil/Documentation.md [hunk 1]
```diff
@@ -0,0 +1 @@
+A measurement of the ability of a fluid to flow through a rock. Represented by the variable "k". The quality of the soil that enables water to move through the profile. Permeability is measured as the number of metres per sec that water moves through the saturated soil.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/h/HydraulicPermeability_3iagtDsw9DTgUKGeH1bJil/Documentation.md]: This does not imply `IfcVolumeMeasure`.

File: IFC4x3/Properties/t/Transmissivity_2iOpQUtFb9RODz2KOn3Z6g/Documentation.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Rate of flow of groundwater under a unit hydraulic gradient through an aquifer of unit width and unit thickness. That is, transmissivity is the product of hydraulic conductivity and thickness of the aquifer. The unit is in m2/day.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/t/Transmissivity_2iOpQUtFb9RODz2KOn3Z6g/Documentation.md]: I did not understand where the "/day" part comes into the unit. It only explains the "m2" part "transmissivity is the product of hydraulic conductivity and thickness of the aquifer".

File: IFC4x3/Properties/h/HydraulicPermeability_3iagtDsw9DTgUKGeH1bJil/Documentation.md (hunk 1 above)
Comment (larswik) [reply IFC4x3/Properties/h/HydraulicPermeability_3iagtDsw9DTgUKGeH1bJil/Documentation.md]: > This does not imply `IfcVolumeMeasure`.

I agree. I had it as IfcVelocityMeasure, but that was changed to IfcVolumeMeasure. Wrong type or wrong description? @j-weil.

Comment (larswik) [reply IFC4x3/Properties/h/HydraulicPermeability_3iagtDsw9DTgUKGeH1bJil/Documentation.md]: IfcVelocityMeasure.

File: IFC4x3/Properties/t/Transmissivity_2iOpQUtFb9RODz2KOn3Z6g/Documentation.md (hunk 2 above)
Comment (larswik) [reply IFC4x3/Properties/t/Transmissivity_2iOpQUtFb9RODz2KOn3Z6g/Documentation.md]: changed description to m2/second.

File: IFC4x3/Properties/h/HydraulicPermeability_3iagtDsw9DTgUKGeH1bJil/Documentation.md (hunk 1 above)
Comment (SergejMuhic) [reply IFC4x3/Properties/h/HydraulicPermeability_3iagtDsw9DTgUKGeH1bJil/Documentation.md]: Permeability is not velocity. I still think we should consider an IfcPermeabilityMeasure.

"IfcLinearVelocityMeasure is a measure of the velocity of a body measured in terms of distance moved per unit time."

Comment (larswik) [reply IFC4x3/Properties/h/HydraulicPermeability_3iagtDsw9DTgUKGeH1bJil/Documentation.md]: This also affects #626 (PermeabilityHoz/PermeabilityVert + Transmissivity)

**Pull request merged**